.github/assert-deployed-bytecode.js

```diff
@@ -0,0 +1,104 @@
+const { AssertionError } = require('chai')
+const chalk = require('chalk')
+const { web3 } = require('hardhat')
+const { readJSON } = require('../scripts/helpers/fs')
+const { APPS_TO_NAMES, CONTRACTS_TO_NAMES, IGNORE_METADATA_CONTRACTS } = require('./deployed-bytecode-consts')
+
+function compareSymbolWise(first, second, ignoreMetadata) {
+  let compareEnd = first.length
+  if (ignoreMetadata) {
+    compareEnd = first.indexOf('a264')
+  }
+  for (var i = 0; i < compareEnd; i++) {
+    if (first[i] != second[i] && first[i] != '0' && second[i] != '0') {
```

Comment:
Maybe it's too dirty for dealing with uninitialized immutable fields, though will work :thinking: 
I mean, immutable fields should have 32 bytes-wide alignment and maybe it could be a good idea to check that zeros are grouped together.

Reply:
Fixed